integr8ly/integreatly-operator
# MGDAPI-4146 Upgrade Keycloak to v18.0.0

# Issue link
https://issues.redhat.com/browse/MGDAPI-4146

# What
- Bumped keycloak version to v18.0.0

# Verification steps
- [Verify upgrading RHOAM](https://docs.google.com/document/d/1P30xs2ReimrXM3xryIn6EYn3cufhep_-BzeEtNfLZDc/edit?usp=sharing) using this branch installs keycloak-operator v18.0.0-legacy
  - `quay.io/bkeane0/managed-api-service-index:1.23.0` (built from master)
  - `quay.io/bkeane0/managed-api-service-index:1.23.1` (built from this PR)
- Keycloak container image should match [this](https://catalog.redhat.com/software/containers/rh-sso-7/sso75-openshift-rhel8/611418806e1e42ca4d6decf1?push_date=1653308451000&architecture=amd64&tag=7.5-27&container-tabs=overview) - **rh-sso-7/sso75-openshift-rhel8**
- Keycloak init container image should match [this](https://catalog.redhat.com/software/containers/rh-sso-7/sso7-rhel8-init-container/6160610d5ac6c12504902068?tag=7.5-14&architecture=amd64&push_date=1653308451000&container-tabs=overview) - **rh-sso-7/sso7-rhel8-init-container**
